### DISCUSSION TITLE
Python Support

### DISCUSSION THREAD

**Discussion by esauser** [DataDog/datadog-for-intellij-platform#14] [Q&A] [answered]
Any plans to support Python?  Is so, roughly when?

**Comment by jfree** [comment, marked as answer]
Yes, it is on our roadmap.  You can expect to see a public beta in the marketplace before the end of Q4.

**Reply by esauser** [reply]
How can I ensure I'll be notified of that?

**Reply by jfree** [reply]
The best I can think of right now is that you subscribe to the releases on this repo.  Choose `Custom` from the watch/unwatch selector near the top of the main page, then check the `Releases` item:

![image](https://github.com/DataDog/datadog-for-intellij-platform/assets/1835893/d2d2959d-3c73-43ea-a9bd-4324687f2539)

You'll get notified of all public releases then you can check the release notes to see if we've added `PyCharm` to the list of supported IDEs.

**Comment by abrooksv** [comment]
We have released our first public beta with PyCharm support to the JetBrains marketplace, please let us know if you find any issues with it.
